src/main/java/pt/teamloan/model/PostingEntity.java

```diff
@@ -102,6 +103,12 @@
 	@Size(max = 4000)
 	private String notes;
 
+	@Enumerated(EnumType.STRING)
+	private ClosePostingReason closePostingReason = ClosePostingReason.NONE;
```

Comment:
I'd suggest two changes:
* change attribute name to "closeReason" (it is already on the posting entity)
* change ClosePostingReason enum name to PostingCloseReason